DPKI_operations.md

```diff
@@ -0,0 +1,38 @@
+# Chapter on PKI/DPKI operations
+
+
+## Introduction to PKI operations:
+In the public key infrastructure (PKI), there are two main operations, which are: **signing request**, and **revocation request**. 
+There is also another operation that is nothing more than a composition of revocation and signing, namely updating.
+
+## Certificate signing request (CSR):
+A CSR is a message from an entity to a trusted authority of the PKI, which contains information about the entity's identity and it is called *certification request information*. Specifically, it contains the name of the entity, the public key of the entity, and optionally a set of attributes. All of that information is signed with the private key of the entity.
+
+A certification request consists of three parts: 
+- certification request information. 
+- a signature algorithm identifier.
```

Comment:
I would use "a" on each line or none

Reply:
oh, the first line can stay as is since information is plural. I would still say "a digital signature..." on the next line